scripts/configurator/parser/cmake_parser.py

```diff
@@ -0,0 +1,37 @@
+from cmake_row import cmake_row, configurator_type, row_type
+from typing import List
+
+def read_file(filepath):
+   with open(filepath) as f:
+      return f.readlines()
+
+def lines_to_rows(file_lines: List[str]) -> List[cmake_row]:
+   """
+   turns the lines of a file (parsed previously) to a list of cmake_rows,
```

Comment:
MISLEADING COMMENT: the parsing is what happens where, previously we just split the file in lines

Reply:
Fixed.